usaspending_api/download/helpers/elasticsearch_download_functions.py

```diff
@@ -52,7 +52,7 @@ def _get_download_ids_generator(cls, search: Union[AwardSearch, TransactionSearc
             yield results
 
     @classmethod
-    def _get_download_ids(cls, filters: dict, size: int = 100000) -> QuerySet:
+    def _get_download_ids(cls, filters: dict, size: int = 10000) -> QuerySet:
```

Comment:
Smaller chunks for downloads? Was an issue encountered or perhaps this is a performance improvement?

Reply:
Intended this to be 10,000 but I added an additional 0 apparently and so this is fixing the mistake. It is also a performance boost.